### PR TITLE
Featured/refactoring cleanup part 2

### PR DESCRIPTION
**Pull request by AD9018**
## Description

Deleted Appelette cases from the site AND implemented the functionality to search multiple data points in any criteria. 
The user is now able to search multiple points as mentioned, for example, the user can now search for asylum cases and really narrow down results and enhancing results by searching for what they want with no constraints no matter how many cities they might want to search. 

Implementing the search functionality was easier than implementing the removal of a data point from the ANT search filter function inside of the table. 

Fixes # (issue)

Fixes restrictive ability for users where only one data point can be searched, deletion of appellate cases as well which we may implement again in the future. 

## Type of change

Change mostly in part to wanted functionality by stakeholders and deletion of appellate cases per stakeholders request as well. 

The next step is refactoring the case table component giving us more performant code and easier for the next cohort to understand. As of now, it is not so easy to comb through the code in a quick manner. 

- [x] New feature (non-breaking change which adds functionality

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes
https://www.loom.com/share/f21ba2f5343e4c1ea232432e459fe150

### CODE REVIEW

**Review by ZacharyCooremans — APPROVED** [review]
Very good job with refactoring and  describing your changes. I would suggest next time to only focus on required components that need to be changed. For example I would have waited until your start implementing your search feature on the judge table before moving the judge columns to utils. In the end good description and video. Keep up the amazing work.

**Review by joshuascan — APPROVED** [review]
Great work on this! This definitely adds to the overall functionality/user experience of the site. The only thing I would say is to just stay within the scope of the pull request and create separate pull requests as needed. Refactoring generally should not change any of the external behavior; it should only be cleaning up and improving the internal structure of the code. The removal of the appellate cases and the revision to the search filters are more outward facing changes.

One last thing I noticed is that it looks like you might be working off of an older version of main as the bar at the top is white instead of blue, and the upload case button is blue instead of orange, among some other differences I noticed. Make sure to pull the most recent version of main before creating a new branch to work on a new ticket. This will help to prevent future merge conflicts. I could be totally wrong about the reasons for the differences though and if I am, just ignore me!

Keep up the great work!

**Review by schandrupatla — APPROVED** [review]
Great work! Definitely,  multiple filters is a great feature for any user. I appreciate the video with details and clarity. I have a suggestion though.
Suggestion: As a new user he wouldn't know he has an option for a multi-filter feature and needs a comma to use it. I would suggest having either a tooltip or information saying how to use that feature . Eg: we show when we are choosing a password. Its more user-friendly that way.

File: src/utils/filter_keyword_utils.js
Comment (chrisjs87) [inline src/utils/filter_keyword_utils.js:1]: Looks like the importing of react was made obsolete for this given it is a utility, you may just want to delete this comment

Comment (chrisjs87) [reply src/utils/filter_keyword_utils.js:1]: Furthermore, I would check each utility you made, and determine if the importing of react is necessary or not. I am not 100% either way. Just worth thinking about!

**Review by chrisjs87 — DISMISSED** [review]
Amazing job overall! Code is cleanly organized, and relevant comments for the most part in your code help to understand what the code is doing. I would ask that you review your utilities and determine if the importing of react is necessary or not for each, as it is commented out in one file, but kept in others. The files in which importing react is kept, I can't tell if it is actually used. However functionally, the code looks fantastic!

**Review by paulstgermain — CHANGES_REQUESTED** [review]
While functionally this code is amazing and nails the functionality we want, there is nearly no clean way to integrate it into the codebase because it stems from a 2-month-old branch - therefore it's missing a lot of the work that has gone into the main production branch. Requesting changes to block this from being merged pending further discussion on the matter.

**Review by paulstgermain — APPROVED** [review]

File: src/utils/filter_keyword_utils.js
Comment (AD9018) [reply src/utils/filter_keyword_utils.js:1]: Funny enough it would not work unless I had imported React, I am not sure why but maybe it has to do with it being a util? We should definitely revisit this another time though. Thanks for the heads-up, still do not really understand why I had to import it to get it to work.

**Comment by paulstgermain** [comment]
Approving changes after debugging session with @AD9018. All git merges are fixed, and all 101 past commits successfully merged with all functionality maintained. 

As a rare case, I'll be merging this PR now and dropping a more cohesive set of feedback after the fact in an effort to unblock this repo!

**Pull request merged**
